tests/pw/pages/sellerVacationPage.ts

```
@@ -0,0 +1,29 @@
+import { Page } from '@playwright/test';
+import { VendorPage } from '@pages/vendorPage';
+import { selector } from '@pages/selectors';
+import { data } from '@utils/testData';
+
+// selectors
+const settingsVendor = selector.vendor.vStoreSettings;
+
+export class SellerVacationPage extends VendorPage {
+    constructor(page: Page) {
+        super(page);
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)